pkg/ephem/fs.go

```diff
@@ -0,0 +1,60 @@
+package ephem
+
+import (
+	"encoding/json"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/moby/sys/mountinfo"
+)
+
+var specialFsRegex = regexp.MustCompile(`^(/proc|/dev|/sys|/run|/var/lib/docker|/var/lib/nfs/rpc_pipefs).*`)
```

Comment:
It's not entirely clear for me what's a "special" filesystem here. Makes sense to filter `/proc`, `/sys`, etc. as we are looking for block devices containing file systems. But why `/var/lib/docker`? Too me, this seems a bit too specific for this layer.